### DISCUSSION TITLE
Shutdown a specific kafka cluster within kubernetes

### DISCUSSION THREAD

**Discussion by pmontagna** [strimzi/strimzi-kafka-operator#8954] [Q&A] [answered]
I have 2 kafka clusters deployed on my Kubernetes cluster:
poc-cluster
poc-cluster2

Per the docs this is how to delete a Kafka cluster:  kubectl -n kafka delete $(kubectl get strimzi -o name -n kafka)

I want to just shutdown poc-cluster.  Using the kubectl command as suggested by the docs would wipe everything except the strimzi-cluster-operator.

**Comment by scholzj** [comment, marked as answer]
That command deletes all Strimzi resources. If the docs got you thinking that it deletes one cluster, please share the exact place so that we can fix it.

If you want to delete just one cluster, you can do it using kubectl as you would with any other Kubernetes resource. So for example `kubectl delete kafka poc-cluster`.

**Reply by pmontagna** [reply]
Thanks, that worked perfectly!  As far as the documentation for managing clusters.   I found this:  
You can combine this strimzi command with other commands. For example, you can pass it into a kubectl delete command to delete all resources in a single command.

kubectl delete $(kubectl get strimzi -o name)

kafka.kafka.strimzi.io "my-cluster" deleted
kafkatopic.kafka.strimzi.io "kafka-apps" deleted
kafkauser.kafka.strimzi.io "my-user" deleted
Deleting all resources in a single operation might be useful, for example, when you are testing new Strimzi features.

In this section : [(https://strimzi.io/docs/operators/latest/deploying.html#management-tasks-str)] in the Resource categories subsection of the documentation.  I could not find what you suggested.   I would be helpful if you can add a Delete cluster subsection.

Thanks

**Reply by scholzj** [reply]
Well, this seems to be fine. It says that it will delete all resources. So that seems sufficiently clear to me. Not sure if deletion of a single resource is mentioned anywhere. Any thoughts @PaulRMellor?
